src/core/myapp.cpp

```diff
@@ -120,6 +120,7 @@ void MyApp::OnEventLoopEnter(wxEventLoopBase *	loop)
 			stopwatch.Start();
 			DoCalculation();
 			total_milliseconds_spent_on_threads += stopwatch.Time();
+      MyInteractiveProgramCleanup();
```

Comment:
Is this meant to be overwritten by programs?

Reply:
Yes, the IMPLEMENT_APP macro that creates the main() function also creates constructors and destructors. The ProgramSpecificInit() then works like a constructor, but for whatever reason the ProgramSpecificCleanup that was already defined is not reached (in MyApp::OnExit) which I think is due to the exit() functions that are all over the place. Somewhat more vexingly, it seems that the ProgramSpecificCleanip() is not override(d, en?)